### DISCUSSION TITLE
macOS readme portion is not self-explanatory

### DISCUSSION THREAD

**Discussion by XantheFIN** [OpenHantek/OpenHantek6022#328] [Ideas]
You have macOS 10.13.6. The application requires macOS 11.6 or later.

**Comment by Ho-Ro** [comment]
> You have macOS 10.13.6. The application requires macOS 11.6 or later.

Please explain more detailed - where do I have 10.13.6?

If you refer to this section 
~~The bundle build doesn't work at the moment ... leave it to a volunteer to get the package build working again.~~
it was already solved with #314 in version 3.3.1. I will update the readme the next days.

**Comment by XantheFIN** [comment]
I didn't spot in readme in front page what is macOS requirement. 
Maybe i move to higher macOS in near future as its hard for ready built macOS open source stuff be build for this instead just compile myself as i have all linux compiling stuff and so freaking much tools running this side i am not just myself yet doing that easily.. Though all major software houses still do release for this platform very well.

I do understand some put own limits where go... but anyway cut my nonsense rumbling i just notified there was no min requirement text anywhere so i felt frustrated after try so wanted just suggest add text there what is supported (by who ever does keep up macOS builds).

**Reply by XantheFIN** [reply]
Ops yes i just copy pasted error message and didn't spot wording so thats my bad if got confused by just that if didn't spot that it was just error message.
